docs: add info on how to use a specific config (#211)
AtomLinter/linter-markdown: README.md

[![Plugin installs!](https://img.shields.io/apm/dm/linter-markdown.svg)](https://atom.io/packages/linter-markdown)
[![Package version!](https://img.shields.io/apm/v/linter-markdown.svg?style=flat)](https://atom.io/packages/linter-markdown)

> Lint markdown files using [remark-lint][remark-lint] and the [linter][linter] package for atom.

Reviewer: ```suggestion
Lint markdown files using [remark-lint][remark-lint] and the
[linter][linter] package for Atom.
```
I'd prefer the line splitting to be in there, it makes no difference when rendered but makes diffs _much_ nicer to view.

Also "atom" should be capitalized.

> ## standard install (default configuration)

Reviewer: ```suggestion
## Standard Installation
```
Headers should always be capitalized, I'd also leave off the part about the configuration as it's described in detail below.

> To use this `linter-markdown` package for Atom, simply install the package using the Atom package manager. If there is no configuration found for **remark-lint**, this linter runs [remark-preset-lint-consistent][consistent] and [remark-preset-lint-recommended][recommended] (both can be turned off). You can a also turn on [remark-preset-lint-markdown-style-guide][styleguide]. By default, [YAML frontmatter][yaml] is supported, but you can turn that off.

Reviewer: Please just add the sentence at the start, the rest is identical and should be left on individual lines.

Author: Ok, fixed.

> If you need a specific configuration for the package, there is 2 ways of installing this `linter-markdown` package: a system-wide way, and a folder-specific way.

Reviewer: 🔥 How `linter-markdown` is installed doesn't change regardless of how the extensions for `markdown` are installed.

Author: I changed nothing because I don't understand what to do...

> ## system-wide install (system-wide configuration)

Reviewer: ```suggestion
## Global Configuration
```
This entire section should be de-emphasized below the project specific configuration.

> The system-wide way of installing the `linter-markdown` package should be used if you use the same configuration on all your markdown documents. This solution is also convenient if your markdown documents are not part of an npm project (and hence do not have an associated package.json file).

Reviewer: ```suggestion
The global configuration should only be used if you need to use the same
configuration on all your Markdown documents, and they aren't contained within
projects of their own.
```
Global configurations are virtually never the right solution, they should only be used if:
* The files aren't in projects **and**
* The files will _never_ be worked on by anyone else (even if the "other person" is yourself later on in the future)

And as stated before this has nothing to do with how `linter-markdown` itself is installed.

> You can set you system-wide configuration in a `.remarkrc` file that should be saved in `~/.remarkrc`, at the root of your home directory.

Reviewer: ```suggestion
You can set you system-wide configuration in a `.remarkrc` file that should be
saved in `~/.remarkrc`, at the root of your home directory.
```

> You then have to globally install all modules referenced in your `.remarkrc` file, alongside `remark` and `remark-cli` (`npm install` with the `-g` or `--global` flag).

Reviewer: ```suggestion
You then have to globally install all modules referenced in your `.remarkrc`
file, alongside `remark` and `remark-cli` (`npm install` with the `-g` or
`--global` flag).
```

The line break between this sentence and the last should probably also be removed.

> ## folder-specific install (project-specific configuration)

Reviewer: This should be listed before the global instructions, project specific ones are _always_ preferred as they should be committed to the repository and shared by anyone working on it.

Global installations are a last resort that should virtually never actually be used.

Author: Ok, fixed.

> ## folder-specific install (project-specific configuration)

Reviewer: ```suggestion
## Project-specific Configuration
```
Let's emphasize that this is related to the configuration, not the method of installation of the plugins. This also has nothing to do with how `linter-markdown` itself is installed so I don't really like the word "install" in the header here as it could confuse people.

> The folder-specific way of installing the `linter-markdown` package should be used if you use differents configurations depending of each markdown document. This solution is also convenient if your markdown documents are part of an npm project (and hence have an associated package.json file).

Reviewer: ```suggestion
The folder-specific way of configuring `remark` should be used if:
* You are working on the project with other people
* You need repeatable results
* You use need a different configuration for each project
```

> You can set you folder-specific configuration in a `.remarkrc` file that should be saved in each folder of your markdown documents.

Reviewer: ```suggestion
You can set you folder-specific configuration in:
* A `remarkConfig` section in `package.json`
* A `.remarkrc.js` or `.remarkrc` file
These should be placed in your project, see
[remark configuration](https://github.com/unifiedjs/unified-engine/blob/master/doc/configure.md)
for details.
```
The link there can probably be put down with the other links.

> You then have to locally install all modules referenced in your `.remarkrc` file, alongside `remark` and `remark-cli` (`npm install` without the `-g` or `--global` flag). If you do install modules globally, you must either use [nvm][], or have a [`prefix` in your `.npmrc`][prefix].

Reviewer: ```suggestion
You then have to locally install all modules referenced in your `.remarkrc`
file, alongside `remark` and `remark-cli` (`npm install` without the `-g` or
`--global` flag).
```
The removed part is only relevant for a global installation.

> ## other informations

Reviewer: ```suggestion
## More Information
```

> If there *is* configuration for **remark-lint**, through `.remarkrc` files or `remarkConfig` in `package.json`s, this linter works just like [remark-cli][cli].

Reviewer: ```suggestion
If there *is* configuration for **remark-lint**, through `.remarkrc` files or
`remarkConfig` in `package.json`s, this linter works just like
[remark-cli][cli].
```

> See [this tutorial][set-up] on how to set-up npm to work without sudo, which is good practise, and comes with the added benefit that `linter-markdown` can pick up on globally installed modules.

Reviewer: ```suggestion
See [this tutorial][set-up] on how to set-up `npm` to work without `sudo`, which
is good practice, and comes with the added benefit that `linter-markdown` can
pick up on globally installed modules.
```
Misspelling of "practise" -> "practice" and no wrapping.

Read more about configuring [remark-lint][configuration] on its README.

> A similar linter, [linter-remark][], runs all remark plugins, but only when they are configured to run.

Reviewer: Again, please don't undo the line breaks.

Author: Ok, fixed.
